src/index.html

```diff
@@ -3,12 +3,43 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="#">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;1,100&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="#">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo__link">
+        <img class="logo" src="images/logo.png"
+        alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="list__items" >
+          <li class="list__item"><a class="is-active nav-item" href="#">APPLE</a></li>
+          <li class="list__item"><a class="nav-item" href="#">SAMSUNG</a></li>
```

Comment:
<img width="1122" alt="Spaces between tags" src="https://github.com/mate-academy/layout_moyo-header/assets/86789161/3670d5e0-6c1c-46d3-a947-fda375a16556">

Reply:
The same in all other places